webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/NodeConfigData.scala

```diff
@@ -1167,7 +1179,8 @@ class TestNodeConfiguration(
     id,
     "",
     None,
-    ""
+    "",
+    _isEnabled = true
```

Comment:
the directive was disable :face_exhaling: A luck it's not used much (appart for the node configuration build)